### ISSUE TITLE
Graph Neural Network based scheduling : Improved throughput under a generalized interference model. (arXiv:2111.00459v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#12644]
Graph Neural Network based scheduling : Improved throughput under a generalized interference model. (arXiv:2111.00459v1 [eess.SY])<br>
https://ift.tt/31mGU0t<br>
<p>In this work, we propose a Graph Convolutional Neural Networks (GCN) based scheduling algorithm for adhoc networks. In particular, we consider a generalized interference model called the $k$-tolerant conflict graph model and design an efficient approximation for the well-known Max-Weight scheduling algorithm. A notable feature of this work is that the proposed method do not require labelled data set (NP-hard to compute) for training the neural network. Instead, we design a loss function that utilises the existing greedy approaches and trains a GCN that improves the performance of greedy approaches. Our extensive numerical experiments illustrate that using our GCN approach, we can significantly ($4$-$20$ percent) improve the performance of the conventional greedy approach.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
